contracts/Tokens/Prime/README.md

```diff
@@ -0,0 +1,345 @@
+
+$$σ_{i,BTC} =min($9600,\text{ } $2500)$$
+
+
+
+**User Rewards**
+
+$$Rewards_{i, BTC} = 8\times 0.2\times \dfrac{1,200^{0.5}\times 2,500^{0.5}}{744,164}$$
+
+$$Rewards_{i, BTC} = \ 0.00372$$
+
+$$\text{User APY Increase} = \dfrac{0.00372}{0.097} = 3.88\%$$
+
+**Expected Rewards Function**
+
+Venus Prime program rewards will automatically increase as a user increases their XVS stake, as long as the amount staked and market participation fall within the limits outlined in the "Rewards" section above.
+
+<figure><img src="https://github.com/VenusProtocol/venus-protocol-documentation/blob/127301b54fb5aa7048aaa65256615690d2c807fb/.gitbook/assets/apy_graph_transparent_2500_corrected_labels.png" alt=""><figcaption><p><em>Please note that the rewards can vary based on the total market participation and the amount of XVS staked, as illustrated by the formula and example above.</em></p></figcaption></figure>
+
+The graph above demonstrates the relationship between an increased XVS staked amount and its effect on market rewards, assuming a constant participation of $2.5K USD in the BTC supply market. This helps visualize how an increase in the staked amount influences the APY.
+
+### Implementation of the rewards in solidity
+
+There is a global `rewardIndex` and `sumOfMembersScore` variables per supported market. `sumOfMembersScore` represents the current sum of all the prime token holders score. And `rewardIndex` needs to be updated whenever a user’s staked XVS or supply/borrow changes. 
+
+```jsx
+// every time accrueInterest is called. delta is interest per score
+delta = totalIncomeToDistribute / sumOfMembersScore
+rewardIndex += delta
+```
+
+Whenever a user’s supply/borrow or XVS vault balance changes we will calculate the rewards accrued and add them to their account:
+
+* In Comptroller (specifically in the `PolicyFacet`), after executing any operation that could impact the Prime score or interest, we accrue the interest and update the score for the prime user by calling `accrueInterestAndUpdateScore`.
+* In the `XVSVault`, after depositing or requesting a withdrawal, the function `xvsUpdated` is invoked, to review the requirements of Prime holders.
+
+This is how we will calculate the user rewards:
+
+```jsx
+rewards = (rewardIndex - userRewardIndex) * scoreOfUser
+```
+
+Then we will update the `userRewardIndex` (`interests[market][account]`) to their current global values.
+
+## Income collection and distribution
+
+Interest reserves (part of the protocol income) from core pool markets are sent to the PSR ([Protocol Share Reserve](https://github.com/VenusProtocol/protocol-reserve/blob/develop/contracts/ProtocolReserve/ProtocolShareReserve.sol)) contract. Based on configuration a certain percentage of spread income from Prime markets is reserved for prime token holders. The interest reserves will be sent to the PSR every 10 blocks (this can be changed by the community via [VIP](https://app.venus.io/governance)).
```

Comment:
```suggestion
Interest reserves (part of the protocol income) from core pool markets are sent to the PSR ([Protocol Share Reserve](https://github.com/VenusProtocol/protocol-reserve/blob/develop/contracts/ProtocolReserve/ProtocolShareReserve.sol)) contract. Based on configuration a certain percentage of spread income from Prime markets is reserved for Prime token holders. The interest reserves will be sent to the PSR every 10 blocks (this can be changed by the community via [VIP](https://app.venus.io/governance)).
```